README.md

```
@@ -16,38 +18,107 @@ It's impossible to talk about *"mockingbird"* without remembering the song [Mock
+``` clj
+(:require 
+  [mockingbird.components.button :refer [button]]
+  [helix.core :refer [$]])
+;; use it then
+  ($ button)
+
+```
+
+you can pass arguments into our components simply as this:
+
+
```

Markdownlint

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)